labgrid/driver/networkusbstoragedriver.py

```diff
@@ -23,6 +26,21 @@ class NetworkUSBStorageDriver(Driver):
         },
     }
 
+    def _check_available(func): # pylint: disable=no-self-argument
+        """Check if a mass storage device is available as a decorator"""
```

Comment:
Shouldn't we really check the path existence here?

Reply:
We would have to call into the remote machine to check that. I don't think its worth it to check this here, its better to fix the reporter to only export existing paths.